back-end/models/Response.model.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const schema = new Schema({
+    question: { type: Schema.Types.ObjectId, ref: 'Question', required: true },
+    answer: { type: String, required: true },
```

Comment:
in thinking about this more, i'm wondering if we need multiple response models: one where it's one answer (string), one where it's an array of answers for multiple choice. could be an easy work around